Added few Channels

Niyasmkkv opened the pull request:
- Adds Dreamz TV ( Closes #4167 )
- Adds Malabar News ( Closes #4145 )
- Adds M Sign Media ( Closes #4170 )
- Adds Sun News ( Closes #4139 )
- Fixes Rosebowl & ADN Gold
- Ditches some specific quality streams for their adaptive playlist.

Niyasmkkv committed:
Added few Channels

Niyasmkkv commented:
@Dum4G  what went wrong? I'm clueless.

LaneSh4d0w commented:
Few trailing spaces on the channel names.

BellezaEmporium committed:
Update in.m3u

BellezaEmporium committed:
Merge branch 'master' into patch-13

LaneSh4d0w commented on channels/in.m3u:
```diff
@@ -375,6 +381,8 @@ https://a.jsrdn.com/broadcast/447912f76b/+0000/c.m3u8
 #EXTINF:-1 tvg-id="PratidinTime.in" tvg-name="Pratidin Time" tvg-country="IN" tvg-language="Hindi" tvg-logo="" group-title="News",Pratidin Time (360p)
 https://rtmp.smartstream.video/pratidintime/pratidintime/playlist.m3u8
+#EXTINF:-1 tvg-id="PravasiTV.in" tvg-name="Pravasi TV" tvg-country="IN" tvg-language="Malayalam" tvg-logo="" group-title="News",Pravasi TV
+https://5ee50688d7b5d.streamlock.net:444/live/pravasi/playlist.m3u8
```
Doesn't seem to work for me.

Niyasmkkv replied:
It was working fine. Not working now. can you remove it?

LaneSh4d0w replied:
I'll wait and try again tomorrow, might simply be Not 24/7.

Niyasmkkv replied:
Not playing in the source website also. Error Code: 232404)

LaneSh4d0w replied:
Still not working right now. Deleting it.

BellezaEmporium committed:
Merge branch 'master' into patch-13

BellezaEmporium committed:
Some modifications

- Ditched some links for their adaptive versions
- Fixed Rosebowl and ADN Gold by adding user-agent

BellezaEmporium committed:
Update in.m3u

LaneSh4d0w commented:
Took advantage of the fact we were working on the Indian playlist to make some additional modifications.

BellezaEmporium committed:
Merge branch 'master' into patch-13

Dum4G commented:
Since I'm away from the Internet I'll leave a pre-approval for this PR
